### DISCUSSION TITLE
MSYS2: Add '/etc' to 'persist'

### DISCUSSION THREAD

**Discussion by michael-taylor** [ScoopInstaller/Main#4836] [Ideas]
Hello,

Is there any downside to adding MSYS2's /etc dir to the persistent folder?

Also, how good of an idea is it to store my persistent folder to a Git repository so that I can share my Scoop app configs across multiple computers?

Thank you!

**Comment by rashil2000** [comment]
> Is there any downside to adding MSYS2's /etc dir to the persistent folder?

None known, as of now. It's worth a try.

> Also, how good of an idea is it to store my persistent folder to a Git repository so that I can share my Scoop app configs across multiple computers?

I don't think it's a good idea. Persist folder can get very large very quickly, because some applications store runtime data and internal packages there.

**Reply by michael-taylor** [reply]
Thank you for the feedback, @rashil2000.  I just learned about Scoop, and it seems like a great project.
